qiskit/pulse/transforms.py

```diff
@@ -94,21 +102,23 @@ def get_max_calibration_duration():
 
         for time, inst in schedule.instructions:
             for chan in inst.channels:
-                if isinstance(chan, MeasureChannel):
+                if isinstance(chan, chans.MeasureChannel):
```

Comment:
I think this is a good example where `import channels as chans` decreases readability. Importing as `chans` to avoid a name conflict doesn't seem like the best approach. Picture doing this for two lists of channels within one function:
```
chans = [MeasureChannel, DriveChannel]
channels = [DriveChannel(0), AcquireChannel(0)]
for chan in chans:
    for channel in channels:
        print(isinstance(channel, chan)
```
by the time I get to the last line, I'm not really sure how a `channel` is different from a `chan`, because semantically the names are identical.

Reply:
I tried to do this but it simply wasn't possible due to circular imports due to the basebackend importing pulse and being imported as part of qiskit init...